catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -331,4 +334,8 @@ private void closeNotificationBar() {
 			sendBroadcast(it);
 		}
 	}
+
+	public static int getId() {
+		return ATOMIC_INTEGER.incrementAndGet();
+	}
```

Comment:
I can't comment on line #150 (`return notificationId++;`) on GitHub. You've to use 
```java
notificationId = getId();
return notificationId;
```
on line #150.

Please rename the member `ATOMIC_INTEGER`. Maybe `UNIQUE_NOTIFICATION_ID_COUNTER` or something like that. And the method `getId` to `getUniqueNotificationId` or so.